catalog/spatial/csw/spatial-csw-source/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/source/CswSource.java

```diff
@@ -384,7 +384,7 @@ public void refresh(Map<String, Object> configuration) throws SecurityServiceExc
 
         String idProp = (String) configuration.get(ID_PROPERTY);
         if (StringUtils.isNotBlank(idProp)) {
-            cswSourceConfiguration.setId(idProp);
+            setId(idProp);
```

Comment:
The id change looks fine, but it doesn't look like refresh() takes in account a changed url.  The SecureCxfClientFactory needs to be reconstructed with a new url if changed in the config.  I'm fine fixing it as part of this ticket  !fix